src/api/LuaApiProvider.tsx

```diff
@@ -30,7 +32,11 @@ export function LuaApiProvider({ children }: PropsWithChildren) {
         const res = await api.singleReturnLibrary();
         setLuaApi(res);
       } catch (e) {
-        console.error(e);
+        if (isReactRenderable(e)) {
```

Comment:
How about renaming the function
```suggestion
        if (isReactNode(e)) {
```
?  "Renderable" immediately made me think of OpenSpace, and I was a bit confused. Also, seems like we are actually testing if it is a react node, and not a "renderable" (whatever that would mean in react context)